core/common/src/main/java/alluxio/security/authorization/DefaultAccessControlList.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.security.authorization;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Default Access control list for a directory.
+ */
+public class DefaultAccessControlList extends AccessControlList {
+  private AccessControlList mAccessAcl;
```

Comment:
why does this class extend `AccessControlList`, and also have a member variable of `AccessControlList`?

Reply:
it is a reference to the access ACL. It is used to reinitialize the owner/group/other entries in the Default ACL when these entries are removed.

Reply:
I see. Can you add a comment about that?